### PR TITLE
Relocate vue-filters

### PR DESCRIPTION
**Pull request by Crashillo**
## :v: What does this PR do?
Move vue-filters to new folder, and update their imports accordingly.

This PR is part of #3572

### CODE REVIEW

**Review by ferblape — APPROVED** [review]
Easy peasy

**Comment by Crashillo** [comment]
I merged (and fixed) this branch into `staging`, but I prefer to wait for https://github.com/PopulateTools/gobierto/pull/3541. So I gonna label it as blocked meanwhile.

File: app/javascript/gobierto_visualizations/webapp/containers/contracts/Summary.vue [hunk 1]
```diff
@@ -147,14 +147,13 @@
           <h3 class="mt1 graph-title">
             {{ labelAmountDistribution }}
           </h3>
-          <div id="amount-distribution-bars"></div>
+          <div id="amount-distribution-bars" />
```
Comment (ferblape) [inline app/javascript/gobierto_visualizations/webapp/containers/contracts/Summary.vue:150]: Is that syntax valid? 🤯

**Review by ferblape — APPROVED** [review]

File: app/javascript/gobierto_visualizations/webapp/containers/contracts/Summary.vue (hunk 1 above)
Comment (Crashillo) [reply app/javascript/gobierto_visualizations/webapp/containers/contracts/Summary.vue:150]: Indeed, it's recommended for empty tags

**Pull request merged**